libs/models/schema/ui/texts/src/lib/models-schema-ui-texts.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HeroText } from './hero-text';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class ModelsSchemaUiTextsModule {}
+
+//export the interface
+export { HeroText }
```

Comment:
Exports happen in the index file (except declarations).

Reply:
There's still this file, it's not required since it has no declarations to manage. Also check how its been done in other model modules in the codebase. If you can sort this I'll merge the PR